pkg/cmd/server/origin/master.go

```diff
@@ -447,6 +465,21 @@ func (c *MasterConfig) getRequestContextMapper() kapi.RequestContextMapper {
 	return c.RequestContextMapper
 }
 
+// ensureMasterAuthorizationNamespace is called as part of global policy initialization to ensure master namespace exists
+func (c *MasterConfig) ensureMasterAuthorizationNamespace() {
+
+	// ensure that master namespace actually exists
+	namespace, err := c.KubeClient().Namespaces().Get(c.MasterAuthorizationNamespace)
+	if err != nil {
+		namespace = &kapi.Namespace{ObjectMeta: kapi.ObjectMeta{Name: c.MasterAuthorizationNamespace}}
+		kapi.FillObjectMetaSystemFields(api.NewContext(), &namespace.ObjectMeta)
```

Comment:
we cant depend on them if the namespace is managed by an external kube process.

this is less boo-hiss in my book than how the policy stuff is stored.